cmd/entrypoint/main.go

```diff
@@ -210,7 +196,7 @@ func main() {
 			// in both cases has an ExitStatus() method with the
 			// same signature.
 			if status, ok := t.Sys().(syscall.WaitStatus); ok {
-				checkForBreakpointOnFailure(e, breakpointExitPostFile)
+				e.CheckForBreakpointOnFailure()
```

Comment:
Nice, but it seems unrelated to this PR.
Is the reason for making `CheckForBreakpointOnFailure` public is to provide unit test coverage?